frontend/src/metabase/admin/settings/components/widgets/SecretKeyWidget/SecretKeyWidget.tsx

```diff
@@ -7,11 +7,19 @@ import { GenerateButton, SecretKeyWidgetRoot } from "./SecretKeyWidget.styled";
 interface SecretKeyWidgetProps {
   onChange: (token: string) => void;
   setting: any;
+  confirmation?: {
+    header: string;
+    dialog: string;
+  };
 }
 
 const SecretKeyWidget = ({
   onChange,
   setting,
+  confirmation = {
+    header: t`Regenerate embedding key?`,
+    dialog: t`This will cause existing embeds to stop working until they are updated with the new key.`,
+  },
```

Comment:
If we end up refactoring how the embedding version of this is setup, we should not need these default values